src/course-unit/preview-changes/index.tsx

```diff
@@ -24,37 +25,30 @@ export interface LibraryChangesMessageData {
   isVertical: boolean,
 }
 
-const PreviewLibraryXBlockChanges = () => {
+export interface BasePreviewLibraryXBlockChangesProps {
+  blockData?: LibraryChangesMessageData,
+  isModalOpen: boolean,
+  closeModal: () => void,
+  acceptChangesMutation: UseMutationResult,
+  ignoreChangesMutation: UseMutationResult,
+  postChange: () => void,
+}
+
+export const BasePreviewLibraryXBlockChanges = ({
```

Comment:
Nit: It's unclear from the names what's the difference between `BasePreviewLibraryXBlockChanges` and `PreviewLibraryXBlockChanges`. JSdoc comments would also be helpful.

Reply:
@bradenmacdonald Renamed and added documentation as well. JSdoc with typescript files seems to raise warnings.

Reply:
I just mean the most basic JSdoc comments describing what each thing is, not necessarily adding type info etc.